plugins/image-source/image-source.c

```diff
@@ -208,7 +208,8 @@ static void image_source_tick(void *data, float seconds)
 			time_t t = get_modified_timestamp(context->file);
 			context->update_time_elapsed = 0.0f;
 
-			if (context->file_timestamp != t) {
+			if (context->file_timestamp != t ||
+			    !context->if4.image3.image2.image.loaded) {
```

Comment:
Another downside to this approach is that we will continually retry to load the image at the polling interval if the file is missing. This could be avoided if we confirm that `t != -1` before attempting the load.

Reply:
Hm, if it causes log spam or something then I'd say it should be handled.

Reply:
I've updated my PR to check for `t != -1` so that we can avoid attempting the image load in a loop. I'd still prefer the other approach I outlined above, but this is at least better now.